src/appMain/life_cycle.h

```diff
@@ -100,7 +101,7 @@ class LifeCycle {
   hmi_message_handler::HMIMessageHandlerImpl* hmi_handler_;
   hmi_message_handler::HMIMessageAdapter* hmi_message_adapter_;
   media_manager::MediaManagerImpl* media_manager_;
-  resumption::LastState* last_state_;
+  resumption::LastStateImpl* last_state_;
```

Comment:
@LuxoftAKutsan I think you right this change is not necessary, fixed in e5c1141.